ras_backstage/common/filters.py

```diff
@@ -0,0 +1,5 @@
+def filter_collection_exercises(exercises, period):
+    for exercise in exercises:
+        if exercise['name'] == period:
+            return exercise
+    return None
```

Comment:
Linter will probably complain about the last line here being unnecessary.

Reply:
Does a function `return None` as default if you don't hit a return statement? I'll remove that if so

Reply:
Yeah, without an explicit return statement all Python methods and function return None by default.